_includes/lc/syllabus.html

```diff
@@ -20,6 +20,9 @@ <h3 id="syllabus-shell">The Unix Shell</h3>
       <li>Pipes and redirection</li>
       <li>Mining or searching in files</li>
       <li><a href="https://librarycarpentry.github.io/lc-shell/reference">Reference...</a></li>
+      <li><a href="https://github.com/foerstner-lab/Bits_and_pieces_for_the_carpentries_workshops/tree/master/python">Lessons...</a></li>
```

Comment:
Dies ist in der Unterliste `Shell`, sollte aber vermutlich in eine eigene `<div class="col-md-6">Python...`, oder? Von mir aus auch gerne doch in `schedule.html` wie in #13. Die Standard-Lesson können wir auch mit dem Linktext `(Alternative)` dazuverlinken.

Reply:
Ah ja genau, da bin ich wohl irgendwie verrutscht, das sollte zum Syllabus

Reply:
Genau da ist es, aber eben in einer Unterliste.